core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/projection/ProjectionOpDesc.scala

```diff
@@ -36,11 +36,33 @@ class ProjectionOpDesc extends MapOpDesc {
     )
   }
 
+  def checkDuplicate: Boolean = {
+    var alias = new ListBuffer[String]()
+    attributes.foreach(attrName => {
+      alias += attrName.getAliasAsString()
+
+    })
+    val uniqueList = alias.distinct
+    val result = uniqueList.size
+    result == attributes.size
```

Comment:
i don't think this is an efficient way to check for duplicates

Reply:
Apparently they improved the code in the latest master. Now it checks output schema for duplicates already so I don't need to include a check function separately.